src/decaylanguage/dec/dec.py

```diff
@@ -742,7 +756,16 @@ def print_decay_modes(
 
         ls = [(bf, ls_attrs_aligned[idx]) for idx, bf in enumerate(ls_dict)]
         ls.sort(key=operator.itemgetter(0), reverse=(not ascending))
-        norm = sum(bf for bf, _ in ls) if normalize else 1
+        norm = 1
+        if normalize:
+            # Get the largest branching fraction
+            i = -1 if ascending else 0
+            # Either normalize to 1 or normalization_bf
+            norm = (
+                ls[i][0] / normalization_bf
```

Comment:
I'm a bit confused about this line. It seems that if a normalization constant is explicitly specified, the actual normalization would be `<largest_br_frac> / <specified_normalization>`.

What this means is that after normalization, the largest branching fraction will be `<specified_normalization>`? Am I interpreting this correctly?

Naively I'd just write `norm = normalization_bf if normalization_bf != 1 else sum(bf for bf, _ in ls)` but I have forgot the technical details of the code and might be completely wrong on this.

Reply:
I re-read the doc string, and it seems like this is the intended behavior. I think it makes sense to do it like this.

Reply:
As you can see above I added more doc and in particular a bunch of examples.

Yes the code is correct for what it is meant to do. But I agree that it can be a bit misleading because `normalization_bf` is effectively a scaling factor where as setting ` normalize=True` does normalise all numbers to a sum of unity. Those are different things. Maybe I should rather have two separate options `normalize` and `scale`, which cannot be used at the same time? I think I will go for that but am welcome to opinions. Thanks.

Reply:
I think separate the `normalize` and `scale` as you described is more clear indeed.

Reply:
Yeah, seems even better now that it is implemented :-).